common/src/main/java/org/apache/falcon/util/Pair.java

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
There is a org.apache.falcon.Pair class already available. Please reuse the same.